pkg/cmd/server/rhoai-normalizer/controller.go

```diff
@@ -388,7 +386,7 @@ func (r *RHOAINormalizerReconcile) processKFMR(ctx context.Context, name types.N
 							&rm,
 							//TODO deal with multiple versions
 							[]openapi.ModelVersion{*mv},
-							map[string][]openapi.ModelArtifact{mvId: {*ma}},
+							map[string][]openapi.ModelArtifact{mvId: mas},
```

Comment:
and there can be more than one model artifact file, so need to include all of them